folly/Utility.h

```diff
@@ -384,4 +385,13 @@ constexpr auto to_unsigned(T const& t) -> typename std::make_unsigned<T>::type {
   return static_cast<U>(t);
 }
 
+template <class E>
+FOLLY_NODISCARD constexpr auto to_underlying_type(E e) noexcept
+{
+    static_assert(std::is_enum<E>::value, "to_underlying_type requires the"
```

Comment:
Shorter message: `"not an enum type"`.

Reply:
Fair point. I like that over multiline literals anyway. 👍